src/logger.ts

```diff
@@ -1,3 +1,45 @@
+// tslint:disable-next-line:no-implicit-dependencies
+import * as core from 'express-serve-static-core';
 import * as pino from 'pino';
 
 export const logger = pino();
+
+/**
+ * log middleware
+ */
+export function logMiddleware(): core.RequestHandler {
+    const handler = (req: any, res: any, next: core.NextFunction) => {
```

Comment:
hack: typescript won't compile with `Express.Request/Response`, the type declarations don't match up with the actual API :/